woodwork/column_accessor.py

```diff
@@ -446,7 +450,9 @@ def _validate_schema(schema, series):
         raise TypeError("Provided schema must be a Woodwork.ColumnSchema object.")
 
     valid_dtype = schema.logical_type._get_valid_dtype(type(series))
-    if str(series.dtype) != valid_dtype:
+    if str(series.dtype) != valid_dtype and not (
```

Comment:
nit, but we might want to consider just making a convenience method for this.

Reply:
Will do!